app/main.py

```diff
@@ -1,8 +1,40 @@
 class Car:
-    # write your code here
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```

Comment:
Place each argument on a new line including `self` so that there would not be any arguments on the same line with opening and closing parenthesis. It would improve the readability. Apply this change to the  `CarWashStation` class as well